db/schema.rb

```diff
@@ -36,11 +36,12 @@
     t.integer  "gender"
     t.integer  "age"
     t.string   "email"
-    t.datetime "created_at",                 null: false
-    t.datetime "updated_at",                 null: false
-    t.boolean  "engaged",    default: false
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
+    t.boolean  "engaged",      default: false
     t.integer  "status"
```

Comment:
Ja det är en sån där väldigt irriterande grej med Rails migrations. Det är också vanligt att den ändrar ordningen på vissa kolumner. Vi kanske ska lägga in den här? https://github.com/jakeonrails/fix-db-schema-conflicts